staging/kube-prometheus-stack/Chart.yaml

```diff
@@ -21,7 +21,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 44.2.1
+version: 44.2.3
```

Comment:
bumped to 44.2.3 (instead of 44.2.2) just to align kps app version https://github.com/mesosphere/kommander-applications/tree/release-2.5/services/kube-prometheus-stack/44.2.2 which also needs to be bumped to 44.2.3